tests/results/DiffTest.py

```diff
@@ -279,3 +279,23 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_diff_creation(self):
+        uut = Diff(create='create_test.py')
+        self.assertEqual(uut.create, 'create_test.py')
+        self.assertEqual(uut._file, [])
+        uut.add_lines(0, ['line2', 'line3'])
+        self.assertEqual(uut.modified, ['line2', 'line3'])
+
+    def test_empty_diff_creation(self):
+        # Testing creation of a diff of not a new file
+        with self.assertRaisesRegexp(ValueError, 'file_list cannot be None')\
+                as err:
+            Diff()
```

Comment:
ok, I see what is the problem here. The latest revision has the variant you requested, but commit wise, I didn't remove this from here, so that's why!